.gitignore

```diff
@@ -0,0 +1 @@
+log
```

Comment:
good to have that on your global .gitignore just to be save, but no harm in adding it here ...